schema/Mutation.graphql

```diff
@@ -84,4 +84,16 @@ type Mutation {
     Revoke user session
     """
     revokeUserSession(id: ObjectID!, displayNotice: Boolean): Boolean!
+
+    """
+    Create a new life data.
+    """
+    createLife(
+        firstName: String!
+        lastName: String!
+        description: String!
+        birthday: DateTime!
+        hobbies: [String!]!
+        title: String!
```

Comment:
Might be good if we put this in an `input` type instead

Reply:
Fix on new update.